compressor_types.go

```diff
@@ -1,21 +1,24 @@
 package compressor
 
 import (
+	"github.com/open-cluster-management/hub-of-hubs-message-compression/compressors"
+	noop "github.com/open-cluster-management/hub-of-hubs-message-compression/compressors/default"
 	"github.com/open-cluster-management/hub-of-hubs-message-compression/compressors/gzip"
 )
 
-// CompressType is the compressing supported types.
+// CompressionType is the compressing supported types.
 //
-// Supported types: GZip.
-type CompressType string
+// Supported types: Default (no op), GZip.
+type CompressionType string
 
 const (
+	// Default returns a no-op compressor (does nothing).
+	Default CompressionType = "default"
```

Comment:
can we name it NoOp instead of default?
default doesn't say anything.. and btw, who said default behavior is without compression?
maybe we want to use gzip by default?